py/vtdb/vtgatev3.py

```diff
@@ -88,14 +88,12 @@ def _create_req(sql, new_binds, tablet_type, not_in_transaction):
 
 class VTGateConnection(object):
   """This utilizes the V3 API of VTGate."""
-  session = None
-  _stream_fields = None
-  _stream_conversions = None
-  _stream_result = None
-  _stream_result_index = None
+
+  stream_execute_returns_generator = True
```

Comment:
vtgatev3 should go away. Maybe you can do that next instead of having to fix it.

The idea is to have vtgatev2 also implement execute and stream_execute (with no routing parameter).